README.md

```diff
@@ -191,7 +191,7 @@ Silent mode off: `.o-ft-buttons--inverse`
 ### Deprecated
 
 The `--faux` modifier is now deprecated and will be removed in next major release.
-The `.o-ft-buttons__pagination` wasn't following the BEM convention properly and will be removed in next major release. Use `..o-ft-buttons-pagination` instead.
+The `.o-ft-buttons__pagination` wasn't following the BEM convention properly and will be removed in next major release. Use `.o-ft-buttons-pagination` instead.
```

Comment:
I thought I'd rename class names in the next major version.

Unless the two (`o-ft-buttons` and `o-buttons` should coexist)?